# file: C:\Users\ADITI\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\scipy\stats\_relative_risk.py
# hypothesis_version: 6.135.14

[0.0, 0.95, 'control_cases', 'control_total', 'exposed_cases', 'exposed_total']